recipes/dedupe_Levenshtein_search/recipe.yaml

```diff
@@ -0,0 +1,47 @@
+context:
+  name: dedupe_Levenshtein_search
+  version: "1.4.5"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/b7/d0/1c97909871021517f55f63bc7db2fe1bd0febf5472e015d94f934efc6998/${{ name }}-${{ version }}.tar.gz
+  sha256: 11f9cf17bade458d04f8a62470050920545c1614a792d786c7ca462a4ff5224a
+
+build:
+  script: python -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compiler names include 'c', 'cxx' and 'fortran', among others.
+    - ${{ compiler('c') }}
+    - ${{ compiler('cxx') }}
+    - ${{ stdlib('c') }}  # If you need any compiler, add the C standard library ("stdlib") too
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python
+
+tests:
+  - python:
+      imports:
+      - dedupe_Levenshtein_search
```

Comment:
This (+ the name of the conda package, aka `name` in the context) should probably be `Levenshtein_search`

https://github.com/dedupeio/dedupe/blob/54ecfe77d41390da66899596834a2bde3712c966/dedupe/levenshtein.py#L3

Reply:
or – according to the linter and sane taste – let's call the conda package `dedupe-levenshtein-search`. `levenshtein-search` is taken by the original project.